03_launch_mgmt_cluster.sh

```diff
@@ -136,15 +147,22 @@ function apply_bm_hosts() {
 function launch_cluster_api_provider_baremetal() {
     pushd "${CAPBMPATH}"
     if [ "${CAPBM_RUN_LOCAL}" = true ]; then
-      touch capbm.out.log
-      touch capbm.err.log
-      make deploy
-      kubectl scale statefulset cluster-api-provider-baremetal-controller-manager -n metal3 --replicas=0
-      nohup make run >> capbm.out.log 2>> capbm.err.log &
+      if [ "${V1ALPHA2_SWITCH}" == true ]; then
+        touch capbm.out.log
+        touch capbm.err.log
+        make install 
+        nohup make run >> capbm.out.log 2>> capbm.err.log &
+      else 
+        touch capbm.out.log
+        touch capbm.err.log
+        make deploy
+        kubectl scale statefulset cluster-api-provider-baremetal-controller-manager -n metal3 --replicas=0
+        nohup make run >> capbm.out.log 2>> capbm.err.log &
+      fi
     else
       make deploy
```

Comment:
make deploy in v1alpha2 is problematic, please use make install/run for now as we don't have the proper registry and images in place